libr/debug/p/native/windows/windows_debug.c

```diff
@@ -1035,14 +1035,23 @@ int w32_continue(RDebug *dbg, int pid, int tid, int sig) {
 	DWORD continue_status = (sig == DBG_EXCEPTION_NOT_HANDLED)
 		? DBG_EXCEPTION_NOT_HANDLED : DBG_EXCEPTION_HANDLED;
 	dbg->tid = w32_select (dbg, pid, tid);
+	r_io_system (dbg->iob.io, sdb_fmt ("pid %d", dbg->tid));
+
+	// Don't continue with a thread that wasn't requested
+	if (dbg->tid != tid) {
+		return false;
+	}
+
 	if (breaked) {
 		breaked = false;
-		return tid;
+		return false;
 	}
+
 	if (!ContinueDebugEvent (rio->pi.dwProcessId, rio->pi.dwThreadId, continue_status)) {
 		r_sys_perror ("w32_continue/ContinueDebugEvent");
 		return false;
 	}
+
 	return tid;
```

Comment:
return bool and int at the same time? this is wrong

Reply:
There was a call chain depending on this rv, fixed now.